cinema/views.py

```diff
@@ -56,3 +85,68 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    @staticmethod
+    def _params_to_ints(query_string):
```

Comment:
this is a method with duplicate code - normally not a good idea. This can be moved to the helper function (utlis.py or similar)